From 802b2d762ee866007e25fa9838e468133c2f51c2 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 7 Feb 2017 18:27:38 +0800
Subject: [PATCH 1651/5242] MLK-14286-5 video: ADV7535: add build support.

commit  c97ce3b4467c19e84f74703282915fd2facaa1a3 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add build support for ADV7535 kernel driver.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 124ff1d4de5bc93d5bd2c68ac0fcb4f414b199ef)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/Kconfig  |    6 ++++++
 drivers/video/fbdev/mxc/Makefile |    1 +
 2 files changed, 7 insertions(+)

diff --git a/drivers/video/fbdev/mxc/Kconfig b/drivers/video/fbdev/mxc/Kconfig
index e69e4e0..d5c439b 100644
--- a/drivers/video/fbdev/mxc/Kconfig
+++ b/drivers/video/fbdev/mxc/Kconfig
@@ -86,6 +86,12 @@ config FB_MXC_DCIC
 	depends on MXC_IPU_V3 || FB_MXS
 	select VIDEOMODE_HELPERS
 
+config FB_MXC_ADV7535
+	tristate "ADI ADV7535 support"
+	depends on I2C
+	help
+	  Driver support for the ADV7535 DSI-to-HDMI module
+
 config HANNSTAR_CABC
 	tristate "Hannstar CABC function"
 	help
diff --git a/drivers/video/fbdev/mxc/Makefile b/drivers/video/fbdev/mxc/Makefile
index 0056b69..a910a3a 100644
--- a/drivers/video/fbdev/mxc/Makefile
+++ b/drivers/video/fbdev/mxc/Makefile
@@ -7,6 +7,7 @@ obj-$(CONFIG_FB_MXC_TRULY_PANEL_TFT3P5581E)	+= mxcfb_hx8363_wvga.o
 obj-$(CONFIG_FB_MXC_LDB) += ldb.o
 obj-$(CONFIG_FB_MXC_HDMI)			+= mxc_hdmi.o
 obj-$(CONFIG_FB_MXC_EDID)			+= mxc_edid.o
+obj-$(CONFIG_FB_MXC_ADV7535)			+= adv7535.o
 obj-$(CONFIG_FB_MXC_SYNC_PANEL) += mxc_dispdrv.o mxc_lcdif.o mxc_ipuv3_fb.o
 obj-$(CONFIG_FB_MXC_EINK_PANEL)			+= mxc_epdc_fb.o
 obj-$(CONFIG_FB_MXC_EINK_V2_PANEL)		+= mxc_epdc_v2_fb.o
-- 
1.7.9.5

